src/mixins/legal-api-mixin.ts

```diff
@@ -47,20 +48,65 @@ export default class LegalApiMixin extends Vue {
         throw error
       })
   }
+  private mockResponse = [
```

Comment:
Make sure you delete this before your final commit :)

Reply:
haha yes :) Thanks!